fal/views/teams.py

```diff
@@ -0,0 +1,204 @@
+from dataclasses import dataclass
+import configparser
+
+from fal.clients.mfalncfm_main import session_scope
+from fal.models import Team, Season, TeamWeeklyAnime, Anime
+
+from typing import Dict, Tuple, List, Mapping, Sequence, TextIO, TYPE_CHECKING
+from sqlalchemy import func
+from sqlalchemy.orm import Session
```

Comment:
it doesn't look like you use this or TYPE_CHECKING

Reply:
I'll remove the imports for TYPE_CHECKING and Session